cmd/vcert/utils.go

```diff
@@ -187,28 +187,9 @@ func fillCertificateRequest(req *certificate.Request, cf *commandFlags) *certifi
 
 		data := strings.Split(validDays, "#")
 		days, _ := strconv.ParseInt(data[0], 10, 64)
-		hours := days * 24
+		duration := time.Duration(days) * time.Hour * 24
 
-		req.ValidityHours = int(hours)
-
-		issuerHint := ""
-		if len(data) > 1 { //means that issuer hint is set
```

Comment:
HI @inteon ,

Why are you removing the issuer hint?

Reply:
I removed the hint because in vcert cli/ vcert Go library, we don't want our users to worry about what issuer is configured in TPP. The great thing about the Venafi control plane is that we can switch the issuers in the control plane without causing any issues for all the applications that want to obtain a machine identity through our API.

Reply:
Making the user-facing API issuer-specific defeats the purpose of a unified control plane in my opinion and will definitely cause problems for my use case.